.github/workflows/eks.yaml

```diff
@@ -100,7 +100,7 @@ jobs:
 
       - name: Install Cilium
         run: |
-          cilium install ${{ steps.vars.outputs.cilium_install_defaults }}
+          cilium install ${{ steps.vars.outputs.cilium_install_defaults }} --encryption
```

Comment:
Looking at the commit message, and since we are already testing with encryption enabled line 141, I suppose this is a temporary commit.

Reply:
yes, but how do I make it run?

Reply:
> which you need to enable by uncommenting lines 11-13

Reply:
Sorry, I edited while you were answering.

Reply:
aha thanks.